packages/react-components/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -65,7 +65,7 @@ export const useFluentProvider_unstable = (
   const { styleTagId, rule } = useFluentProviderThemeStyleTag({
     theme: mergedTheme,
     targetDocument,
-    rendererAttributes: renderer.styleElementAttributes ?? {},
+    rendererAttributes: renderer.styleElementAttributes ?? EMPTY_OBJECT,
```

Comment:
I think we should introduce a new empty_object, not reuse the current one to make sure we not sharing memory with other components